src/tests/ftest/security/container_security_acl.py

```diff
@@ -57,32 +69,37 @@ def test_container_user_acl(self):
             (7)Verify container permission d, delete
             (8)Cleanup
 
-        :avocado: tags=all,full_regression,security,container_acl,cont_user_sec
+        :avocado: tags=all,full_regression,security,container_acl,
+        :avocado: tags=cont_user_sec,cont_group_sec,cont_sec
         """
 
         #(1)Setup
         self.log.info("(1)==>Setup container user acl test.")
-        base_acl_entries = [
-            "",
-            secTestBase.acl_entry("user", self.current_user, ""),
-            secTestBase.acl_entry("group", "GROUP", ""),
-            secTestBase.acl_entry("group", self.current_group, ""),
-            secTestBase.acl_entry("user", "EVERYONE", "")]
-        user_type = self.params.get("user_type",
-                                    "/run/container_acl/*", "user")
         cont_permission, expect_read, expect_write = self.params.get(
             "perm_expect", "/run/container_acl/permissions/*")
-        test_user = self.params.get("new_user", "/run/container_acl/*")
-        test_group = self.params.get("new_group", "/run/container_acl/*")
+        new_test_user = self.params.get("new_user", "/run/container_acl/*")
+        new_test_group = self.params.get("new_group", "/run/container_acl/*")
         attribute_name, attribute_value = self.params.get(
             "attribute", "/run/container_acl/*")
         property_name, property_value = self.params.get(
             "property", "/run/container_acl/*")
-        secTestBase.add_del_user(self.hostlist_clients, "useradd", test_user)
-        secTestBase.add_del_user(self.hostlist_clients, "groupadd", test_group)
+        secTestBase.add_del_user(
+            self.hostlist_clients, "useradd", new_test_user)
+        secTestBase.add_del_user(
+            self.hostlist_clients, "groupadd", new_test_group)
         acl_file_name = os.path.join(
             self.tmp, self.params.get(
                 "acl_file_name", "/run/container_acl/*", "cont_test_acl.txt"))
+        test_user = self.params.get(
+            "testuser", "/run/container_acl/daos_user/*")
+        test_user_type = secTestBase.get_user_type(test_user)
+        base_acl_entries = self.get_base_acl_entries(test_user)
+        if test_user == "user":
+            test_user = self.current_user
+        if test_user == "group":
+            test_user = self.current_group
+        self.log.info(
+            "==>(1.1)Start testing container acl on user: %s", test_user)
```

Comment:
Should this output indicate if a group is being tested? Based on the code above.

Reply:
Thanks for the comment.
There are 5 different test_user (OWNER, user, user-group, GROUP and EVERYONE), which defined from the test.yaml file. The log.info just provides info for debugging.

Reply:
I figured it was for debugging. I was just thinking in the case where the user/group names are the same, it might be easier to figure out what's going on if the type was specified in the message (user or group).